backend/audit/cross_validation/test_auditee_ueis_match.py

```diff
@@ -0,0 +1,56 @@
+from django.test import TestCase
+
+from audit.models import SingleAuditChecklist
+
+from .auditee_ueis_match import auditee_ueis_match
+from .sac_validation_shape import sac_validation_shape
+
+from model_bakery import baker
+
+
+class AuditeeUEIsMatchTests(TestCase):
+    def test_all_sections_none(self):
```

Comment:
**comment**: These test names are reasonable, but I think that they could still use docstrings that explain the different situations they are trying to test, e.g.
```suggestion
    def test_all_sections_none(self):
         """For a SAC with all sections empty, the auditee UEIs match"""
```